### ISSUE TITLE
OE3 b2 - RPi - CEC broken

### ISSUE THREAD

**Issue by vriesm** [OpenELEC/OpenELEC.tv#1556]
CEC seems to be broken in OE3b2 in combination with a Philips TV.

The problem is easy to reproduce, however hard to explain, but  I will try:

The TV remote control works, however if e.g. the navigation buttons are used in the menu, it only registers one time to the right, while more times the right button is pressed.

If between two right pressed another key is pressed e.g. up, then it will move again to the right, so:

right -> right -> right in the menu results in just one step to the right

right  -> up -> right in the menu results in two steps to the right

So, in short, pressing the same key multiple times are handeld as pressing this key once.
